babel.config.js

```diff
@@ -183,6 +183,20 @@ function bool(value) {
   return value && value !== "false" && value !== "0";
 }
 
+// A minimum semver GTE implementation
+// Limitation:
+// - it only supports comparing major and minor version, assuming Node.js will never ship
+//   features in patch release so we will never need to compare a version with "1.2.3"
+//
+// @example
+// semverGte("8.10", "8.9") // true
+// semverGte("8.9", "8.9") // true
+// semverGte("9.0", "8.9") // true
+// semverGte("8.9", "8.10") // false
+// TODO: figure out how to inject it to the `@babel/template` usage so we don't need to
+// copy and paste it.
+// `((v,w)=>(v=v.split("."),w=w.split("."),+v[0]>+w[0]||v[0]==w[0]&&+v[1]>=+w[1]))`;
```

Comment:
You could to

```js
const semverGte = template.ast`(v,w)=>(v=v.split("."),w=w.split("."),+v[0]>+w[0]||v[0]==w[0]&&+v[1]>=+w[1])`
```

And then use it with interpolations in the other templates.
```

Reply:
Ok it looks like I was wrong, because we are not using `template.ast` here but the version that returns a factory function 😅